Automattic/wp-calypso
# Add Balan and Pollard site designs to Gutenboarding

Apply diff D55054-code before testing creating a site so that you have access to the annotations and the template/demo preview for these two designs.

This PR adds in the new Balan and Pollard site designs (#48381)

#### Changes proposed in this Pull Request

* Add `balan` and `pollard` new site designs

__Note:__ the screenshots of these short designs don't work very well with the layout for picking a design in Gutenboarding, so another solution for the screenshots / layout for including these sorts of designs might be required. E.g. here's how they currently look:

| Balan | Pollard |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/14988353/103978924-b7956200-51d0-11eb-9568-928d575e175b.png) | ![image](https://user-images.githubusercontent.com/14988353/103978888-a77d8280-51d0-11eb-8977-a535bb112438.png) |

#### Testing instructions



* Apply D55054-code and sandbox the public-api
* Go to /new and at the design step you should see the new Balan and Pollard site designs
* You should be able to create a new site with each of these designs, and the newly created site should look like the designs

Fixes #48381
